src/net/RTP/RTPSession.cs

```diff
@@ -1469,14 +1470,20 @@ public SDPMediaFormat GetSendingFormat(SDPMediaTypesEnum mediaType)
             }
         }
 
+
+        private ushort NextSeqNum(ushort seqNum)
+        {
+            return (ushort)(++seqNum & 0xffff);
+        }

```

Comment:
Even though it gets handled it'd be better to avoid the overflow.

````
ushort seqNum = UInt16.MaxValue;
checked
{ 
	seqNum = (ushort)(++seqNum & 0xffff);
}
````

This one liner was deliberately duplicated in each method to avoid the overhead of a method call for each RTP packet. It is probably better to make it into a method as you have done as the C# compiler will most likely inline it anyway and even if not the overhead of a method call will be way less than the network IO.

Reply:
I will mark the method with 'aggressiv inlining' so it will force the compiler to inline it.

Reply:
And change to the equivalent of the below to avoid the overflow?

````
videoTrack.SeqNum = (videoTrack.SeqNum == UInt16.MaxValue) ? (ushort)0 : (ushort)(videoTrack.SeqNum + 1);
````

Reply:
Added a unit test to confirm that the 'longer' version is actually faster, so I will change it too